docker/mint/mint.Dockerfile

```diff
@@ -18,10 +18,11 @@ FROM toolchain AS builder
 
 COPY . /libra
 
-RUN cargo build --release -p libra-node -p client && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p client -p dynamic-config-builder && cd target/release && rm -r build deps incremental
```

Comment:
This will miss the docker layer cache.  I think we can land this as is to unblock and fix the caching in another diff.

Reply:
what's the purpose in performing the deletion?

Reply:
not sure i fully grok the concern here, where else would we build this?